examples/phi3v_example.py

```diff
@@ -0,0 +1,64 @@
+import os
+import subprocess
+
+from PIL import Image
+from transformers import AutoProcessor
+
+from vllm import LLM, SamplingParams
+from vllm.multimodal.image import ImagePixelData
+
+
+def run_phi3v():
+    model_path = "microsoft/Phi-3-vision-128k-instruct"
+    processor = AutoProcessor.from_pretrained(model_path,
+                                              trust_remote_code=True)
+    llm = LLM(
+        model=model_path,
+        trust_remote_code=True,
+        image_input_type="pixel_values",
+        image_token_id=-1,
+        image_input_shape="1,3,1008,1344",
+        image_feature_size=1024,
+        disable_image_processor=False,
+    )
+
+    image = Image.open("images/stop_sign.jpg")
+    user_prompt = "<|user|>\n"
+    assistant_prompt = "<|assistant|>\n"
+    suffix = "<|end|>\n"
+
+    # single-image prompt
+    prompt = "What is shown in this image?"
+    prompt = f"{user_prompt}<|image_1|>\n{prompt}{suffix}{assistant_prompt}"
+
+    inputs = processor(prompt, image, return_tensors="pt")
```

Comment:
With the addition of image processor, it should no longer be necessary to add all this boilerplate since the image processor is supposed to be loaded internally by vLLM.